webflux6-micrometer/src/main/resources/application.yaml

```diff
@@ -0,0 +1,42 @@
+management:
+  # only expose /health from actuator
+  endpoints:
+    enabled-by-default: false
+    web:
+      base-path: ''
+      path-mapping:
+        health: /health
+  endpoint:
+    health:
+      enabled: true
+
+  tracing:
+    brave:
+      # Note: There is no property to bind ${brave.traceId128Bit:false}
+      span-joining-supported: ${brave.supportsJoin:true}
+    sampling:
+      # There doesn't seem to be a way to skip tracing /health by property
```

Comment:
I'm probably missing something as it seems like no one would want to trace /health by default, and especially not have a way to disable it..

<img width="1122" alt="Screenshot 2024-02-06 at 20 50 56" src="https://github.com/openzipkin/brave-example/assets/64215/dd8f7a90-a9a0-4648-bac0-6e4c5efc12c9">